templates/taxi/manufacturer_list.html

```diff
@@ -2,16 +2,18 @@
 
 {% block content %}
     <h1>Manufacturer List
-    </h1>
+    <a href="{% url 'taxi:manufacturer-create' %}">+</a>
```

Comment:
Consider adding a 'Cancel' button next to the 'Delete' button as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'